src/applications/soundDetectionDemo/spinnakerEventsMapper/src/vSpinnakerEventsMapper.cpp

```diff
@@ -0,0 +1,235 @@
+//
+// Created by Daniel Gutierrez-Galan
+// dgutierrez@atc.us.es
+// University of Seville
+// 19/Nov/2020
+//
+
+#include <yarp/os/all.h>
```

Comment:
what's the difference between cochlea events mapper and spinnaker events mapper? One is input and the other is output? I would suggest having these combined in a single application as I can imagine they are very dependent on one another that a change in one might affect the other. Therefore having them together makes sense.